src/Metrics/DefaultMetricsHandler.php

```diff
@@ -38,6 +38,9 @@ private function getOrCreateBucket(): MetricsBucket
         $bucket = null;
         if ($cache->has(CacheKey::METRICS_BUCKET)) {
             $bucket = $cache->get(CacheKey::METRICS_BUCKET);
+            if (is_string($bucket)) {
```

Comment:
Quick double check - this should allow a user to replace an existing version connected to a cache that still has the old format and still work?

Reply:
Yes, that's exactly its purpose, before this it would return the object itself, now it returns the serialized string.

Reply:
Fantastic!